pulser/tests/test_json.py

```diff
@@ -57,6 +61,23 @@ def test_register_3d():
     assert reg == encode_decode(seq).register
 
 
+def test_layout():
+    custom_layout = RegisterLayout([[0, 0], [1, 1], [1, 0], [0, 1]])
+    new_custom_layout = encode_decode(custom_layout)
+    assert new_custom_layout == custom_layout
+    assert type(new_custom_layout) is RegisterLayout
+
+    tri_layout = TriangularLatticeLayout(100, 10)
+    new_tri_layout = encode_decode(tri_layout)
+    assert new_tri_layout == tri_layout
+    assert isinstance(new_tri_layout, TriangularLatticeLayout)
```

Comment:
Why not also using`type(…) is …` here?

Reply:
No good reason, I'll change it for consistency's sake.